scripts/apps/users/views/user-preferences.html

```diff
@@ -89,6 +89,9 @@
                 <label>
                     {{ :: preferences['desks:preferred'].label | translate}}
                   </label>
+                  <p class="discreet" translate>
+                    By selecting the desks as your preferred desks, they appear first in the order when sending or publishing items.
+                  </p>git 
```

Comment:
Is this `git` test intended? Looks out of place. Ignore my message if I'm wrong.